### PR TITLE
fix: undo redo property pane border focus

### LINKED ISSUE #14735: [Bug]: Undo Redo flashing is not visible anymore
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

When we do `Ctrl + z`, we used to get a yellow flash on the property control where the undo is applied.
Looks like that is no longer available.

### Steps To Reproduce

Make some changes to any widget's property and press `Ctrl + z`

### Public Sample App

_No response_

### Version

release

### PR DESCRIPTION
**Pull request by yaldram**
## Description
- When we do undo after changing the Properties from the Property Pane we should scroll to the element and show a yellow border (basically focus on the element).

Fixes #14735

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- Tested Manually on the canvas.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/client/src/sagas/ReplaySaga.ts [hunk 1]
```diff
@@ -119,9 +119,13 @@ export function* openPropertyPaneSaga(replay: any) {
...
     }
 
+    /**
+     * Property Updates - ['widgets', 'hluiy8se0g', 'menuColor']
+     * We take the 1st and 2nd element to generate the element id.
+     */
     flashElementsById(
       btoa(
-        replay.widgets[replayWidgetId].propertyUpdates.slice(0, 2).join("."),
+        replay.widgets[replayWidgetId].propertyUpdates.slice(1, 3).join("."),
```
Comment (jsartisan) [inline app/client/src/sagas/ReplaySaga.ts:128]: How was this working before? Double-check this, please.

Comment (rahulramesha) [reply app/client/src/sagas/ReplaySaga.ts:128]: @yaldram, as @jsartisan mentioned, could you check what changed

Comment (yaldram) [reply app/client/src/sagas/ReplaySaga.ts:128]: Yeah I did triarge this issue, I think the ordering might have changed. The above changes are only affecting the flashing while undo, so there won't be any other side-effects for sure. Also our goal was to create the same ids as the property pane controls so we get it from the 1 and 2nd elements.

**Review by rahulramesha — APPROVED** [review]
LGTM

**Review by albinAppsmith — APPROVED** [review]
Approving ADS changes

**Pull request closed without merging**